py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2694,6 +2695,13 @@ def sort_validator(node: Node, settings: Optional[CompilationSettings] = None) -
 
 
 def topk_sort_validator(k: int) -> bool:
+
+    # topk layer supports dynamic k value but we cannot determine supported dynamic topk value at
+    # compile time.
+    if k == DYNAMIC_DIM:
+        _LOGGER.debug("k value cannot be dynamic!")
```

Comment:
Can you add a bit more detail here? Remember that in the logs it wont be immediately obvious what is printing this message. 

So something like 

`[top_k validator] Converter does not support k being a dynamic value, therefore aten::topk will run in PyTorch`